### PR TITLE
Update rides.js

### PR DESCRIPTION
**Pull request by kpots**
text amends

### CODE REVIEW

File: packages/website/src/pages/rides.jsx [hunk 1]
```diff
@@ -21,19 +21,26 @@ class RidesPage extends Component {
...
-          <h3>Training Rides</h3>
-          <p>Depending on the time of year and people's training goals, there may be a faster group on Saturdays and sometimes Sundays too.</p>
-          <p>These rides are 100-200km in length.</p>
+          <p>We will be restarting group rides after 29th March 2021 in line with the latest <a href='https://www.britishcycling.org.uk/about/article/20200512-about-bc-news-British-Cycling-Updated-Coronavirus-Guidance-0'>British Cycling Covid-19 guidance</a> published 9th March in groups of 6-8 people. The first Social Ride will be Saturday 3rd April.</p>
+          <p>If you would like to join a ride before then, we are still riding in pairs so drop us a line via <a href='mailto:peckhamcc@gmail.com?subject = Find me a ride'>email</a> or social media and we will try to match you up with another rider.</p>
```
Comment (kpots) [inline packages/website/src/pages/rides.jsx:25]:
```suggestion
          <p>If you would like to join a ride before then, we are still riding in pairs so <Link to='/contact'>drop us a line</Link> or contact us via social media and we will try to match you up with another rider.</p>
```

File: packages/website/src/pages/rides.jsx [hunk 2]
```diff
@@ -21,19 +21,26 @@ class RidesPage extends Component {
...
+          <h3>How to join a ride</h3>
+          <p>Looking to join your first ride with Peckham Cycle Club? We look forward to welcoming you!</p>
+          <p>Every Saturday we run a Social Ride aimed at introducing new riders to the Club that starts at 8am (summer months) or 8.30am (winter months) at <a href='http://www.southwark.gov.uk/libraries/find-a-library?chapter=13'>Peckham Library</a>. This is usually the <a href='https://ridewithgps.com/routes/34683876'>same route</a> each week and heads out through South East London into the lanes of Kent. It’s about 60km and we'll usually be riding for about 3 hours, with a few hills along the way! We ride in groups of 6-8, typically around 20-22kph but don’t worry if you’re not sure whether that speed is right for you: it is always a no drop ride.</p>
+          <p>If you would like to join us please <a href='mailto:peckhamcc@gmail.com?subject = I want to ride my bicycle with you'>email us</a> in advance and let us know which Saturday date you would like to join.</p>
```
Comment (kpots) [inline packages/website/src/pages/rides.jsx:30]:
```suggestion
          <p>If you would like to join us please <Link to='/contact'>contact us</Link> in advance and let us know which Saturday date you would like to join.</p>
```

File: packages/website/src/pages/rides.jsx [hunk 3]
```diff
@@ -21,19 +21,26 @@ class RidesPage extends Component {
...
+          <p>We ride in single or double file and help everyone to stay together by not going too far off the front. We regroup regularly and take breaks for snacks / coffee / loo along the way.</p>
+          <h3>What to bring</h3>
+          <p>We ask that you come prepared with a few things to make your ride easier. Take a look at our <Link to='/equipment'>Equipment</Link>page to find out what to bring along with you </p>
+          <h3>Am I ready?</h3>
+          <p>We would expect new riders to be confident on their bikes, to be able to ride for 2-3 hours in one go, and to usually have a road bike. These aren’t essential but if you’re concerned about the distance, have a hybrid bike or anything else let us know when you email us.</p>
+          <p>We offer a Buddy Ride Scheme aimed at people who might not be quite ready to join the Social. In these rides you will be paired up one-on-one with a PCC member to do a slightly easier route but still get the full PCC experience.  Please <a href='mailto:peckhamcc@gmail.com?subject = Find me a buddy'>email us</a> if this is of interest.</p>
```
Comment (kpots) [inline packages/website/src/pages/rides.jsx:38]:
```suggestion
          <p>We offer a Buddy Ride Scheme aimed at people who might not be quite ready to join the Social. In these rides you will be paired up one-on-one with a PCC member to do a slightly easier route but still get the full PCC experience.  Please <Link to='/contact'>contact us</Link> if this is of interest.</p>
```

**Comment by achingbrain** [comment]
Merged by #37

**Pull request closed without merging**